### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found few hundred misspellings.
This is the first batch.

Would you like me to continue?

### CODE REVIEW

**Review by SamChou19815 — CHANGES_REQUESTED** [review]
If you want to help, please split this into the parts that has user visible behavior change first, and also fix the tests. Otherwise we won't want a change that just runs some automated spell fix tool over all the files.

**Comment by szepeviktor** [comment]
> just runs some automated spell fix tool over all the files.

All right.
I've worked on this PR for two hours using `typos`.

Could you help me where to fix tests? I've never seen OCaml source code and I am not a flow user.

**Comment by szepeviktor** [comment]
> and also fix the tests

I see! In `./tests/type_at_pos_expression/type_at_pos_expression.exp`
How to regenerate that file?

File: src/hack_forked/utils/collections/third-party/flow_map.ml [hunk 1]
```diff
@@ -4,7 +4,7 @@
  *                                                                     *
  *                           Objective Caml                            *
  *                                                                     *
- *            Xavier Leroy, projet Cristal, INRIA Rocquencourt         *
+ *            Xavier Leroy, project Cristal, INRIA Rocquencourt        *
```
Comment (SamChou19815) [inline src/hack_forked/utils/collections/third-party/flow_map.ml:7]: Do not change license header, especially other people's names

**Comment by SamChou19815** [comment]
Run `./runtests.sh -f -b /path/to/compiled/flow` to re-record the tests. You will need to figure out how to compile it, which you can try to follow the build steps in the CI config.

File: src/hack_forked/utils/collections/third-party/flow_map.ml (hunk 1 above)
Comment (szepeviktor) [reply src/hack_forked/utils/collections/third-party/flow_map.ml:7]: Excuse me.

> The Cristal project ended in 2005.

http://cristal.inria.fr/

**Comment by szepeviktor** [comment]
Thank you @SamChou19815 

What should be my next step?

**Comment by szepeviktor** [comment]
I am not able to fix the remaining test.
Would you help me or should I close this PR?

**Comment by szepeviktor** [comment]
Good bye!

**Pull request closed without merging**